### PR TITLE
feat: add classNames prop to `Booker` atom for passing external styles

### PR DESCRIPTION
**Pull request by Ryukemeister**
## What does this PR do?

Currently there's no way to style the booker atom from the oustide. This PR adds a prop called `customClassNames` through which we can pass in external classnames to customize the styling of the booker

### CODE REVIEW

File: packages/features/bookings/Booker/Booker.tsx [hunk 1]
```diff
@@ -268,14 +272,13 @@ const BookerComponent = ({
         <div
           ref={animationScope}
           className={classNames(
-            // Sets booker size css variables for the size of all the columns.
             ...getBookerSizeClassNames(layout, bookerState, hideEventTypeDetails),
-            "bg-default dark:bg-muted grid max-w-full items-start dark:[color-scheme:dark] sm:transition-[width] sm:duration-300 sm:motion-reduce:transition-none md:flex-row",
+            `bg-default ${customClassNames?.bookerContainer} dark:bg-muted grid max-w-full items-start border dark:[color-scheme:dark] sm:transition-[width] sm:duration-300 sm:motion-reduce:transition-none md:flex-row`,
```
Comment (ThyMinimalDev) [inline packages/features/bookings/Booker/Booker.tsx:276]: extra border to remove

File: packages/features/bookings/Booker/Booker.tsx [hunk 2]
```diff
@@ -353,7 +365,7 @@ const BookerComponent = ({
             <BookerSection
               key="book-event-form"
               area="main"
-              className="border-subtle sticky top-0 ml-[-1px] h-full p-6 md:w-[var(--booker-main-width)] md:border-l"
+              className="sticky top-0 ml-[-1px] h-full border border-yellow-500 p-6 md:w-[var(--booker-main-width)] md:border-l"
```
Comment (ThyMinimalDev) [inline packages/features/bookings/Booker/Booker.tsx:368]: border border-yellow-500 to remove

Comment (ThyMinimalDev) [reply packages/features/bookings/Booker/Booker.tsx:368]: no className here ?

File: packages/features/bookings/Booker/Booker.tsx [hunk 3]
```diff
@@ -375,6 +398,7 @@ const BookerComponent = ({
               visible={layout === BookerLayouts.WEEK_VIEW}
               className="border-subtle sticky top-0 ml-[-1px] h-full md:border-l"
               {...fadeInLeft}>
+              This is the large calendar
```
Comment (ThyMinimalDev) [inline packages/features/bookings/Booker/Booker.tsx:401]: to remove

File: packages/features/bookings/Booker/Booker.tsx [hunk 4]
```diff
@@ -390,14 +414,15 @@ const BookerComponent = ({
                 layout === BookerLayouts.COLUMN_VIEW
               }
               className={classNames(
-                "border-subtle rtl:border-default flex h-full w-full flex-col overflow-x-auto px-5 py-3 pb-0 rtl:border-r ltr:md:border-l",
+                `border-subtle flex h-full w-full flex-col overflow-x-auto px-5 py-3 pb-0 rtl:border-r ltr:md:border-l`,
```
Comment (ThyMinimalDev) [inline packages/features/bookings/Booker/Booker.tsx:417]: revert if nothing is different

File: packages/platform/examples/base/src/pages/booking.tsx [hunk 5]
```diff
@@ -16,7 +16,7 @@ export default function Bookings(props: { calUsername: string; calEmail: string
   const eventTypeSlugQueryParam = (router.query.eventTypeSlug as string) ?? "";
   return (
     <main
-      className={`flex min-h-screen flex-col ${inter.className} main text-default flex min-h-full w-full flex-col items-center overflow-visible`}>
+      className={`flex min-h-screen flex-col bg-white ${inter.className} main text-default flex min-h-full w-full flex-col items-center overflow-visible`}>
```
Comment (ThyMinimalDev) [inline packages/platform/examples/base/src/pages/booking.tsx:19]: remove bg-white

File: packages/features/bookings/Booker/Booker.tsx (hunk 4 above)
Comment (ThyMinimalDev) [reply packages/features/bookings/Booker/Booker.tsx:417]: no className here ?

File: packages/features/bookings/Booker/Booker.tsx [hunk 6]
```diff
@@ -250,6 +253,7 @@ const BookerComponent = ({
         <div
           className="animate-fade-in-up fixed bottom-2 z-40 my-2 opacity-0"
           style={{ animationDelay: "1s" }}>
+          This is the booking tag manager
```
Comment (ThyMinimalDev) [inline packages/features/bookings/Booker/Booker.tsx:256]: this should be removed I think

File: packages/features/bookings/Booker/Booker.tsx [hunk 7]
```diff
@@ -64,7 +64,10 @@ const BookerComponent = ({
   verifyCode,
   isPlatform,
   orgBannerUrl,
+  customClassNames,
 }: BookerProps & WrappedBookerProps) => {
+  console.log(customClassNames, " custom classnames from booker");
```
Comment (ThyMinimalDev) [inline packages/features/bookings/Booker/Booker.tsx:69]: remove console log

File: packages/features/bookings/Booker/components/AvailableTimeSlots.tsx [hunk 8]
```diff
@@ -114,6 +126,8 @@ export const AvailableTimeSlots = ({
           slotsPerDay.map((slots) => (
             <div key={slots.date} className="scroll-bar h-full w-full overflow-y-auto overflow-x-hidden">
               <AvailableTimes
+                className={`scroll-bar w-full overflow-y-auto overflow-x-hidden ${customClassnames?.availableTimeSlotsContainer}`}
```
Comment (ThyMinimalDev) [inline packages/features/bookings/Booker/components/AvailableTimeSlots.tsx:129]: why do we have these new classess added here ? I mean this: scroll-bar w-full overflow-y-auto overflow-x-hidden

Comment (Ryukemeister) [reply packages/features/bookings/Booker/components/AvailableTimeSlots.tsx:129]: My bad we previously didn't had the outside div and everything was being passed in the `AvailableTimes` className prop, might have inserted that at the time of resolving merge conflicts

**Review by ThyMinimalDev — APPROVED** [review]

**Pull request merged**